src/crocoddyl_motion_server_node.cpp

```diff
@@ -128,36 +128,40 @@ namespace panda_torque_mpc
             auto collision_model = boost::make_shared<pinocchio::GeometryModel>();
             collision_model = loadPandaGeometryModel(model_pin_);
 
-            double radius = 0.35/2.0;
+            std::cout << "parser in motion" << std::endl;
```

Comment:
```suggestion
            std::cout << "Parsing obstacles" << std::endl;
```
This makes this command more descriptive. If something is in motion, this would mean the process will take some more time than few micro seconds